src/read.c

```diff
@@ -98,32 +95,30 @@ extern inline struct ReaderState * ReaderState(void)
 **  is <Statements>. The functions 'ReadExpr' and 'ReadStats' must therefore
 **  be declared forward.
 */
-static void ReadExpr(ScannerState * s, TypSymbolSet follow, Char mode);
+static void ReadExpr(ReaderState * rs, TypSymbolSet follow, Char mode);
```

Comment:
From here on, we see a lot of changes identical to this: instead of just a `ScannerState`, we now pass around a `ReaderState` (which contains a `ScannerState`).